### PR TITLE
Fix permissions on rpm.egg and remove cron installer

### PR DESCRIPTION
**Pull request by SteveHNH**
Remove the cron installation from the RPM so we can move it to the register steps. Also fix permissions on rpm.egg so they are world readable

### CODE REVIEW

**Comment by kylape** [comment]
After checking out your branch, building and installing the RPM, I now also have group write perms on the egg:

```
$ sudo ls -l /etc/insights-client
total 548
-rw-rw----+ 1 root root   4586 Sep 13 10:59 cert-api.access.redhat.com.pem
-rw-rw----+ 1 root root   1144 Sep 13 10:59 insights-client.conf
-rwxrwxr-x+ 1 root root    456 Sep 13 10:59 insights-client.cron
-rw-rw----+ 1 root root   1147 Sep 13 10:59 redhattools.pub.gpg
-rw-rw-r--+ 1 root root 536275 Sep 19 10:10 rpm.egg
-rw-rw-r--+ 1 root root    811 Sep 15 12:50 rpm.egg.asc
```

**Comment by SteveHNH** [comment]
@kylape weird. Take Two! Let me give it another shot.

**Comment by SteveHNH** [comment]
@kylape should be good to go now. I moved some of the permissions stanzas around for readability. Goes from defaultattr down to individually set permissions per file.

File: etc/insights-client.spec [hunk 1]
```diff
@@ -80,11 +80,13 @@ ln -sf %{_bindir}/insights-client %{_bindir}/redhat-access-insights
 if ! [ -d "/etc/redhat-access-insights" ]; then
 mkdir /etc/redhat-access-insights
 fi
+# Symlink new cron job if the old one exists. Remove the old one
```
Comment (kylape) [inline etc/insights-client.spec]: Looks like there's some trailing whitespace in this if block.

**Comment by kylape** [comment]
The permissions look good now!

**Comment by SteveHNH** [comment]
@kylape whitespace removed

**Pull request merged**